### PR TITLE
[EDU-5280] Guide webpage to pdf template

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes
Adds the guide for the Webpage to PDF Resume template

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: How to deploy an online resume using the Webpage to PDF Resume template
+description: >-
+  This template contains the configurations to create a resume for the web, print or PDF using the Webpage to PDF Resume project.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx:4]:
```suggestion
  This template contains the configurations to create a resume for the web, print, or PDF using the Webpage to PDF Resume project.
```

File: src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+permalink: /documentation/products/guides/astro-webpage-to-pdf-resume-template/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+The **Webpage to PDF Resume template** contains the configurations for *creating a resume that can be displayed on the web, printed, or saved as a PDF using the Webpage to PDF Resume project*. Once deployed, you can easily customize it with your personal information and develop your portfolio.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx:17]:
```suggestion
The **Webpage to PDF Resume template** contains the configurations for creating a resume that can be displayed on the web, printed, or saved as a PDF using the Webpage to PDF Resume project. Once deployed, you can easily customize it with your personal information and develop your portfolio.
```

File: src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+---
+
+## Setting up the template
+
+In the configuration form, you must provide the information to configure your application. Fill in the presented fields. 
+
+Fields identified with an asterisk are mandatory.
+
+1. Define a name for your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx:52]: is it missing the GitHub step?

File: src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
+title: Como implantar um currículo online usando o template Webpage to PDF Resume
+description: >-
+  Este template contém as configurações para criar um currículo para a web, impressão ou PDF usando o projeto Webpage to PDF Resume.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx:4]:
```suggestion
  Este template contém as configurações para criar um currículo para a web, para impressão ou PDF usando o projeto Webpage to PDF Resume.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+permalink: /documentation/products/guides/astro-webpage-to-pdf-resume-template/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+O template **Webpage to PDF Resume** contém as configurações para *criar um currículo que pode ser exibido na web, impresso ou salvo como PDF usando o projeto Webpage to PDF Resume*. Uma vez implantado, você pode facilmente personalizá-lo com suas informações pessoais e desenvolver seu portfólio.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx:17]:
```suggestion
O template **Webpage to PDF Resume** contém as configurações para criar um currículo que pode ser exibido na web, impresso ou salvo como PDF usando o projeto Webpage to PDF Resume. Uma vez implantado, você pode facilmente personalizá-lo com suas informações pessoais e desenvolver seu portfólio.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+---
+
+## Configure o template
+
+No formulário de configuração, você deve fornecer as informações para configurar sua aplicação. Preencha os campos apresentados. 
+
+Os campos identificados com um asterisco são obrigatórios.
+
+1. Defina um nome para sua edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx:52]: ídem aqui

File: src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+## Gerencie o template
+
+Considerando que essa configuração inicial pode não ser ótima para sua edge application específica, todas as configurações podem ser personalizadas a qualquer momento que você precise, usando a plataforma da Azion.
+
+Para gerenciar e editar as configurações da sua aplicação edge, siga esses passos:
+
+1. [Acesse o Azion Console](https://console.azion.com).
+2. No canto superior esquerdo, selecione **Products Menu** (o ícone de três linhas horizontais) > **Edge Application**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/webpage-to-pdf-resume.mdx:72]:
```suggestion
2. No canto superior esquerdo, selecione **Products Menu**, o ícone de três linhas horizontais, > **Edge Application**.
```

File: src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+## Managing the template
+
+Considering that this initial setup may not be optimal for your specific edge application, all settings can be customized any time you need by using Azion's platform.
+
+To manage and edit your edge application's settings, follow these steps:
+
+1. [Access Azion Console](https://console.azion.com).
+2. On the upper-left corner, select **Products menu** (the icon with three horizontal lines) > **Edge Application**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/webpage-to-pdf-resume.mdx:72]:
```suggestion
2. On the upper-left corner, select **Products menu**, the icon with three horizontal lines, > **Edge Application**.
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**